lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -260,17 +263,19 @@ def cmd_banner(*args)
   def cmd_tip_help
     print_line "Usage: tip [options]"
```

Comment:
Should this line not be changed to reflect the fact that we added in the `tips` command?

Reply:
Fixed with latest commit